src/app/lib/pages/map_page/map_page.dart

```diff
@@ -163,6 +163,8 @@ class MapPageState extends State<MapPage> {
       city = splitAddress[1].trim();
     }
 
+    city = city.toLowerCase();
```

Comment:
```suggestion
```
all cities stored in firebase are capitalized so this would cause the maps page to try to query a lowercase city that wouldn't exist. This is why custom events arents displaying.

Reply:
wait didn't we make it lowercase last time?

Reply:
we made it capitalized by siraj updated the scraper so we are all good ignore this suggestion

Reply:
we need to update the existing custom events collection though

Reply:
updated custom events